docs/experiments/holdouts.md

```diff
@@ -0,0 +1,88 @@
+# Holdouts
+
+Eppo's Holdout Analysis allows your company to validate the impact of Experimentation.
+
+This functionality offers the ability to set a holdout audience in feature flags that keeps an audience isolated from all active experiments, 
+and measures metric changes for the holdout audience versus an audience that only experiences winning experiments.
+
+## Journey of a Holdout user
+
+Define one or more holdout groups with a relevant key and the desired active date range. You can have multiple holdout groups running at once (e.g one for each team that runs experiments).
+
```

Comment:
The holdout is split into two groups: `status quo` who always see the control experience, and `winning variants` who see the treatment from experiments that are rolled out.